cmd/juju/commands/debuglog.go

```diff
@@ -166,13 +166,13 @@ func (c *debugLogCommand) SetFlags(f *gnuflag.FlagSet) {
 	f.StringVar(&c.level, "l", "", "Log level to show, one of [TRACE, DEBUG, INFO, WARNING, ERROR]")
 	f.StringVar(&c.level, "level", "", "")
 
-	f.UintVar(&c.params.Backlog, "n", defaultLineCount, "Show this many of the most recent (possibly filtered) lines, and continue to append")
-	f.UintVar(&c.params.Backlog, "lines", defaultLineCount, "")
-	f.UintVar(&c.params.Limit, "limit", 0, "Exit once this many of the most recent (possibly filtered) lines are shown")
-	f.BoolVar(&c.params.Replay, "replay", false, "Show the entire (possibly filtered) log and continue to append")
+	f.UintVar(&c.params.Backlog, "n", 0, "Show this many of the most recent lines and continue to append new ones")
```

Comment:
The default value isn't 0, it's `defaultLineCount` and if we add `0` her ethe help text will be wrong.
I know this was done so we could see if the user had specified anything. What needs to be done is to implement a bespoke `flag.Getter` to record whether it's been set or not.